frontend/dockerfile/dockerfile_test.go

```diff
@@ -1553,13 +1564,19 @@ COPY --from=build /out .
 }
 
 func testGlobalArgErrors(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	f := getFrontend(t, sb)
 
-	dockerfile := []byte(`
+	dockerfile := []byte(integration.UnixOrWindows(
+		`
 ARG FOO=${FOO:?"custom error"}
 FROM busybox
-`)
+`,
+		`
+FROM nanoserver AS build
+ARG FOO
```

Comment:
not looked into this yet, but why would you not just do the same as lx here?
```
ARG FOO=${FOO:?"custom error"}
```

Reply:
It seems there's no native support for that kind of syntax on Windows containers at the moment. I've created an issue to further investigate this item [#5428](https://github.com/moby/buildkit/issues/5428).

I've reverted this change so that we can investigate this further.